<html>
<head>
<link rel="stylesheet" type="text/css" href="./css/main.css" />
<title>Nina Luhmann</title>
<meta name="author" content="Nina Luhmann">
<meta name="keywords" content="Nina Luhmann, Mainz University, UniversitÃ¤t Mainz, angewandte Bioinformatik, Studienfachberatung, Bielefeld University, Warwick University, Comparative Genomics, PhySca, Genome Informatics, DiDy">
<meta name="description" content="Nina Luhmann, Mainz University" />
</head>

<style>
  .bottom {
     margin-bottom: 5px;
  }
</style>

<script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
     (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
     })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
     
     ga('create', 'UA-77395025-1', 'auto');
     ga('send', 'pageview');
     
    </script>



<div id="main_page">

<body bgcolor="white" text="#191919">

<font size="2">
<font face="Helvetica Neue, Helvetica, Liberation Sans, Arial">
    <br>
<font size="+1">
    <center>
<!-- <a href="http://uni-bielefeld.de/%28en%29/index.html" target="_blank">Bielefeld University</a> -->
<a href="https://github.com/nluhmann" target="_blank">GitHub</a>
&nbsp; - &nbsp;
    
<!-- <a href="http://gi.cebitec.uni-bielefeld.de" target="_blank">Genome Informatics</a> -->
<a href="https://scholar.google.ca/citations?user=JOMIaYYAAAAJ&hl=en" target="_blank">Google Scholar</a>
&nbsp; - &nbsp;

<a href="https://www.linkedin.com/in/nina-luhmann-87830b102/" target="_blank">LinkedIn</a>
&nbsp; - &nbsp;

<a href="https://twitter.com/nina_luhmann" target="_blank">Twitter</a>
&nbsp; - &nbsp;


    
 <a href="https://orcid.org/0000-0001-8112-7027" target="_blank">ORCID</a> 
</center>
</font>
    
    
<hr size="1" color="aba28b" noshade>
    


<br><br><br>
<center>
<table width="80%" border="0" cellpadding="10" cellspacing="2" style="margin-left:auto;margin-right:auto">
 <tr>
   <td>
   &nbsp &nbsp &nbsp &nbsp <img src="./pics/nina.jpeg" alt="Me" border="0" width=110px height=180px>
   </td>
   <td>
   <p align="left"><font size="+2">Nina Luhmann</font><br><br>
   <a href="https://personen.uni-mainz.de/public/person/278099" target="_blank">Academic Staff Member Bioinformatics</a><br>
   Institute of Computer Science,<br>
   Mainz University<br><br>
   E-mail: <a href="mailto:nluhmann@uni-mainz.de">nluhmann [at] uni-mainz.de</a><br>
   </td>
 </tr>
</table>
</center>
<br> <br>



</div>




<div id="main_page">


<!-- 
<br><br>

<font size="+1">News</font> <br><br>

I will be attending Genome Informatics 2018 (19.-21.09.), let me know if you will be there!

<br><br>
 -->

<hr size="1" color="aba28b" noshade>
<br> <br>

I am an academic staff member for bioinformatics at the Institute of Computer Science at Mainz University.


Previously I have been a postdoctoral Research Fellow at the University of Warwick in the group of <a href="http://www2.warwick.ac.uk/fac/med/research/biomedical/mi/staff/achtmanlab/" target="_blank">Mark Achtman</a>.<br><br>

In 2016, I obtained my PhD in Bioinformatics working in the <a href="http://gi.cebitec.uni-bielefeld.de" target="_blank">Genome Informatics</a> group at the <a href="https://techfak.uni-bielefeld.de/en/faculty-technology" target="_blank">Faculty of Technology</a> at Bielefeld University. I was a student in the German-Canadian DFG International Research Training Group <a href="http://didy.uni-bielefeld.de" target="_blank">DiDy</a> ("Computational Methods for the Analysis of the Diversity and Dynamics of Genomes"). My thesis project was jointly supervised by <a href="https://www.techfak.uni-bielefeld.de/~stoye/" target="_blank">Jens Stoye</a> (Bielefeld University, Germany), <a href="http://www.cebitec.uni-bielefeld.de/~roland" target="_blank">Roland Wittler</a> (Bielefeld University, Germany) and <a href="https://cchauve.github.io/index.html" target="_blank">Cedric Chauve</a> (Simon Fraser University, Canada).
<br><br>

My main research interest lies in comparative genomics. During my PhD, I worked on methods integrating ancient DNA sequencing data into comparative assembly approaches. 

<br><br>


<hr size="1" color="aba28b" noshade>




<br>  <br>
                             
<font size="+1">Publications</font> <br><br>

<a href="https://pub.uni-bielefeld.de/person/37807704" target="_blank">PUB </a> | <a href="https://scholar.google.de/citations?user=JOMIaYYAAAAJ&hl=en" target="_blank">Google Scholar</a>

<br><br>
2018
<br>
<ul><li>
<a href="https://genome.cshlp.org/content/28/9/1395.short" target="_blank"> GrapeTree: Visualization of core genomic relationships among 100,000 bacterial pathogens </a><br>
Zhou Z, Alikhan N, Sergeant M, Luhmann N, Vaz C, Francisco A, Carrico J, Achtman M (2018) <br>
Genome research, 28(9), 1395-1404.
<li>
<a href="https://ieeexplore.ieee.org/abstract/document/8316878/" target="_blank"> Scaffolding of Ancient Contigs and Ancestral Reconstruction in a Phylogenetic Framework (Journal version) </a><br>
Luhmann N, Chauve C, Stoye J, Wittler R <br>
IEEE/ACM Transactions on Computational Biology and Bioinformatics.
<li>
<a href="https://link.springer.com/chapter/10.1007/978-3-319-89929-9_15" target="_blank"> Accurate Reconstruction of Microbial Strains from Metagenomic Sequencing Using Representative Reference Genomes </a><br>
Zhou Z, Luhmann N, Alikhan NF, Quince C, Achtman M (2018) <br>
In: Raphael B. (eds) Research in Computational Molecular Biology. RECOMB 2018. Lecture Notes in Computer Science, vol 10812. Springer.
<a href="https://www.biorxiv.org/content/early/2017/11/09/215707" target="_blank"> (Preprint) </a>
<li>
<a href="https://doi.org/10.1007/978-1-4939-7463-4_13" target="_blank"> Comparative Methods for Reconstructing Ancient Genome Organization </a><br>
Anselmetti Y, Luhmann N, B&eacute;rard S, Tannier E, Chauve C (2018) <br>
In: Comparative Genomics: Methods and Protocols. Setubal JC, Stoye J , Stadler P (Eds); Springer New York: 343-362.
</ul>
<br>
2017
<br>
<ul>
<li>
<a href="http://doi.org/10.1099/mgen.0.000123" target="_blank"> Comparative scaffolding and gap filling of ancient bacterial genomes applied to two ancient Yersinia pestis genomes </a><br>
Luhmann N, D&ouml;rr D, Chauve C (2017) <br>
Microbial Genomics, 3.
<li>
<a href="https://pub.uni-bielefeld.de/publication/2909213" target="_blank"> Phylogenetic assembly of paleogenomes integrating ancient DNA data </a><br>
Luhmann N (2017) <br>
Bielefeld University.
<li>
<a href="http://doi.org/10.1109/TCBB.2017.2661761" target="_blank"> The SCJ Small Parsimony Problem for Weighted Gene Adjacencies (Journal version) </a><br>
Luhmann N, Lafond M, Th&eacute;venin A, Ouangraoua A, Wittler R, Chauve C (2017) <br>
IEEE/ACM Transactions on Computational Biology and Bioinformatics 14.
</ul>
<br>
2016
<br>
<ul>
<li>
<a href="http://doi.org/10.1007/978-3-319-38782-6_17" target="_blank"> The SCJ small parsimony problem for weighted gene adjacencies </a><br>
Luhmann N, Th&eacute;venin A, Ouangraoua A, Wittler R, Chauve C (2016) <br>
In: Proceedings 12th International Symposium, ISBRA 2016. Lecture Notes in Computer Science, 9683. Springer International Publishing: 200-210.
</ul>
<br>
2014
<br>
<ul>
<li>
<a href="http://doi.org/10.1007/978-3-319-12418-6_17" target="_blank"> Scaffolding of Ancient Contigs and Ancestral Reconstruction in a Phylogenetic Framework </a><br>
Luhmann N, Chauve C, Stoye J, Wittler R (2014) <br>
In: Proc. of BSB 2014. S&eacute;rgio C (Ed); LNBI, 8826. Springer Verlag: 135-143.
</ul>



<br><br>




<hr size="1" color="aba28b" noshade>


<br>  <br>



<font size="+1">Software</font>  <br><br>

See <a href="https://github.com/nluhmann/" target="_blank">Github</a> repository.



<br>
<br>


<hr size="1" color="aba28b" noshade>


<br>  <br>



<font size="+1">Teaching</font>  <br><br>

Bielefeld University:
<ul>
<li> "Sequenzanalyse - Meilensteine und aktuelle Themeninhalte" (student seminar in sequence analysis) [2015/2016]
<li> "Algorithmische Implementierung" (programming of bioinformatics algorithms) [2015/2016]
<li> "Vorlesung Sequenzanalyse" (sequence analysis lecture) [2014/2015]
<li> "Sequenzanalyse-Praktikum" (sequence analysis practical course) [2014]
<li> Teaching assistance in Algorithms and Datastructures, Sequence analysis I+II, Basics in computer science, "Vorkurs Informatik" [2010-2013]
</ul>







</div>

</font></font>


<br><br><br><br><br>
<br><br><br><br><br>
<br><br><br><br><br>
<br><br><br><br><br>

<br><br><br><br><br>
<br><br><br><br><br>


</body>

</html>
